### PR TITLE
Update pl/java docs for JVM version, shared library loc, CREATE EXTEN…

### PR DESCRIPTION
**Pull request by dyozie**
…SION command, .sql file deprecation

These changes (except for JVM version) will be merged into into 5X_STABLE as well.

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/extensions/pl_java.xml [hunk 1]
```diff
@@ -226,13 +226,12 @@
...
-                runtime libraries. PL/Java depends on <codeph>libjvm.so</codeph> and the shared
-                object should be in your <codeph>LD_LIBRARY_PATH</codeph>. By default,
-                  <codeph>libjvm.so</codeph> is available in
-                  <codeph>$JAVA_HOME/jre/lib/amd64/server</codeph>. This example adds the directory to
-                the environment
-                variable.<codeblock>LD_LIBRARY_PATH=$GPHOME/lib:$GPHOME/ext/python/lib:<b>$JAVA_HOME/jre/lib/amd64/server</b>:$LD_LIBRARY_PATH</codeblock></li>
+              <li>Set the <codeph>LD_LIBRARY_PATH</codeph> to include the directory with the Java
+                server runtime libraries. PL/Java depends on <codeph>libjvm.so</codeph> and the
+                shared object should be in your <codeph>LD_LIBRARY_PATH</codeph>. By default,
+                  <codeph>libjvm.so</codeph> is available in <codeph>$JAVA_HOME/lib/server</codeph>.
```
Comment (gfphoenix78) [inline gpdb-doc/dita/ref_guide/extensions/pl_java.xml]: The directories of libjvm.so in JDK 8 and JDK 11 are different.

| JDK version | directory of libjvm.so | 
|---|---|
| 8 | $JAVA_HOME/jre/lib/amd64/server |
| 11 | $JAVA_HOME/lib/server |

File: gpdb-doc/dita/ref_guide/extensions/pl_java.xml [hunk 2]
```diff
@@ -248,22 +247,20 @@
...
     <topic id="topic8" xml:lang="en">
       <title id="pv165472">Remove PL/Java Support for a Database</title>
       <body>
-        <p>For a database that no long requires the PL/Java language, remove support for PL/Java.
-          Run the <codeph>uninstall.sql</codeph> file as the gpadmin user. For example, this command
-          disables the PL/Java language in the specified database. </p>
-        <codeblock>$ psql -d mydatabase
-  -f $GPHOME/share/postgresql/pljava/uninstall.sql</codeblock>
+        <p>Use the <codeph>DROP EXTENSION</codeph> command to remove support for PL/Java from a
+          database. For example, this command disables the PL/Java language in "testdb". </p>
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/extensions/pl_java.xml]: codeph testdb here?

File: gpdb-doc/dita/ref_guide/extensions/pl_java.xml [hunk 3]
```diff
@@ -248,22 +247,20 @@
...
         </li>
-        <li id="pv165686">
-          <xref href="#topic9" type="topic" format="dita"/>
+        <li id="pv165686"><xref href="#topic9" type="topic" format="dita"/>
         </li>
       </ul>
     </body>
     <topic id="topic8" xml:lang="en">
       <title id="pv165472">Remove PL/Java Support for a Database</title>
       <body>
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/extensions/pl_java.xml]: interesting that we mention "DROP EXTENSION" here, but introduce "CREATE EXTENSION" in a later section.

Comment (dyozie) [reply gpdb-doc/dita/ref_guide/extensions/pl_java.xml]: I agree - found this confusing myself when editing the file.  Not sure how this organization came to be but we should seriously consider changing it.

File: gpdb-doc/dita/ref_guide/extensions/pl_java.xml (hunk 1 above)
Comment (dyozie) [reply gpdb-doc/dita/ref_guide/extensions/pl_java.xml]: Thanks - I added the defaults for each version.

File: gpdb-doc/dita/ref_guide/extensions/pl_java.xml (hunk 2 above)
Comment (dyozie) [reply gpdb-doc/dita/ref_guide/extensions/pl_java.xml]: Done.

File: gpdb-doc/dita/ref_guide/extensions/pl_java.xml (hunk 3 above)
Comment (dyozie) [reply gpdb-doc/dita/ref_guide/extensions/pl_java.xml]: I moved the uninstall after the "enabling" section.

**Pull request merged**